content/specs-macos/xcode-15-1.md

```diff
@@ -1,19 +1,22 @@
 ---
-description: A list of tools available out-of-the-box on Codemagic macOS Apple silicon build machines using Xcode version 15.1.
-title: macOS Apple silicon build machine specification (Xcode 15.1+)
+title: Xcode 15.1.x
 aliases:
 
-weight: 13
+weight: 109
 ---
 
+Codemagic offers multiple build machines with different specifications and pre-installed tools. You can choose between them by specifying the required Xcode version.
```

Comment:
Here and elsewhere, it would make sense to mention instance/machine type as another variable. We can link to https://docs.codemagic.io/yaml-basic-configuration/yaml-getting-started/#instance-type
>You can choose between them by specifying the required Xcode version and instance type.

Reply:
I don't think it's a good idea - we should keep focus on macOS and general available instance type.